src/clarity-angular/datagrid/datagrid-column.ts

```diff
@@ -4,32 +4,44 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {
-    Component, ContentChild, HostBinding, Input, Output, EventEmitter, ElementRef, ViewChild
+    Component,
+    ContentChild,
+    HostBinding,
+    Input,
+    Output,
+    EventEmitter,
+    ElementRef,
+    ViewChild,
+    AfterViewInit
 } from "@angular/core";
 import {Subscription} from "rxjs";
 
-import {DatagridPropertyComparator} from "./built-in/comparators/datagrid-property-comparator";
-import {DatagridPropertyStringFilter} from "./built-in/filters/datagrid-property-string-filter";
-import {Comparator} from "./interfaces/comparator";
-import {CustomFilter} from "./providers/custom-filter";
-import {Sort} from "./providers/sort";
-import {DatagridFilterRegistrar} from "./utils/datagrid-filter-registrar";
-import {FiltersProvider} from "./providers/filters";
-import {DatagridStringFilterImpl} from "./built-in/filters/datagrid-string-filter-impl";
-import {SortOrder} from "./interfaces/sort-order";
-import {DragDispatcher} from "./providers/drag-dispatcher";
+import { DatagridPropertyComparator } from "./built-in/comparators/datagrid-property-comparator";
+import { DatagridPropertyStringFilter } from "./built-in/filters/datagrid-property-string-filter";
+import { Comparator } from "./interfaces/comparator";
+import { CustomFilter } from "./providers/custom-filter";
+import { Sort } from "./providers/sort";
+import { DatagridFilterRegistrar } from "./utils/datagrid-filter-registrar";
+import { FiltersProvider } from "./providers/filters";
+import { DatagridStringFilterImpl } from "./built-in/filters/datagrid-string-filter-impl";
+import { SortOrder } from "./interfaces/sort-order";
+import { DragDispatcher } from "./providers/drag-dispatcher";
+import { DatagridHideableColumn } from "./datagrid-hideable-column";
+
+let nbCount: number = 0;
+
 
 @Component({
     selector: "clr-dg-column",
     template: `
-        <div class="datagrid-column-flex">
+        <div class="datagrid-column-flex" #hideableColumn>
```

Comment:
Yep, leftover for initial prototype / exploration of the solution.
Will remove.